release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -88,6 +88,12 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-networking"]
 === Networking
 
+[id="ocp-4-16-networking-multiple-cidr"]
+==== Define multiple CIDR blocks for one network security group rule
+
+With this release, you can now create multiple CIDR blocks with one network security group (NSG) for Ingress controllers in {product-title} clusters hosted on Azure. You can do this using the `allowedSourceRanges` field. Creating multiple Ingress controllers with the `allowedSourceRanges` field configured will allow those Ingress controllers to share the maximum limit of 4000 rules between them.
```

Comment:
Sure, thanks for writing this up.

Just a nit pick, I think the idea of multiple CIDR blocks in an NSG is a bit of implementation detail in this situation as the end user would probably never see this happening (unless they view the Azure NSGs directly). I think the release note should avoid directly implying that "you" can create multiple CIDR blocks in an NSG, and imply more that it's something that Azure improved and happens in the background when you use `allowedSourceRanges` on ingress controllers.

```suggestion
With this release, IP addresses and ranges are handled more efficiently in network security groups (NSG) for {product-title} clusters hosted on Azure. As a result, the maximum limit of CIDRs for all Ingress Controllers in Azure clusters, using the `allowedSourceRanges` field, has increased from approximately 1000 to 4000 CIDRs.
```

Feel free to wordsmith or adapt this to what make sense, or let me know if you disagree.

Reply:
Excellent, this clears a lot up. Thank you!